sc/social/like/interfaces.py

```diff
@@ -249,3 +249,10 @@ class ISocialLikeSettings(model.Schema):
         required=False,
         default='',
     )
+
+
+class IAdditionalMetatags(Interface):
```

Comment:
I strongly dislike this name, what do you think about `IOGProperties` or something like that?

Reply:
I used a generic name because in some cases one may want to add some non-og metatags. I don't know whether now there is any disalignment betwween OpenGrap and Twitter Card metatags, but at some point there was the need to add some tags that were not specifically OpenGraph.

But anyway, it's just a naming issue, so I can rename it without any problem.